datasets/dbrd/README.md

```diff
@@ -0,0 +1,170 @@
+- 100K<n<1M
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+- text-classification
+task_ids:
+- language-modeling
+- sentiment-classification
+---
+
+# Dataset Card Creation Guide
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [Dutch Book Review Dataset (DBRD) homepage](https://benjaminvdb.github.io/DBRD)
+- **Repository:** https://github.com/benjaminvdb/DBRD
+- **Paper:** [The merits of Universal Language Model Fine-tuning for Small Datasets - a case with Dutch book reviews](https://arxiv.org/abs/1910.00896)
+- **Leaderboard:** 
+- **Point of Contact:** [Benjamin van der Burgh](benjaminvdb@gmail.com)
+
+### Dataset Summary
+
+The DBRD (pronounced *dee-bird*) dataset contains over 110k book reviews along with associated binary sentiment polarity labels and is intended as a benchmark for sentiment classification in Dutch. It was created due to a lack of annotated datasets in Dutch that are suitable for sentiment classification.
+
+### Supported Tasks and Leaderboards
+
+- `sequence-modeling`: The dataset can be used to train a model for sequence modeling, more specifically language modeling.
+- `text-classification`: The dataset can be used to train a model for text classification, more specifically sentiment classification, using the provided positive/negative sentiment polarity labels.
+
+### Languages
+
+Non-Dutch reviews were filtered out using [langdetect](https://github.com/Mimino666/langdetect), and all reviews should therefore be in Dutch (nl). They are written by reviewers on [Hebban](https://www.hebban.nl), a Dutch website for book reviews. Reviews aren't curated and are of varying lengths and qualities, as can be expected.
+
+## Dataset Structure
+
+### Data Instances
+
+The dataset contains three subsets: train, test and unsupervised. The `train` and `test` sets contain labels, while the `unsupervised` set doesn't (the value for label is -1 for each instance in unsupervised).
+
+```
+DatasetDict({
```

Comment:
here we expect to see one example of instance.
For example you can load the dataset and get one by running
```python
dataset["train"][0]
```

Reply:
I've now provided an example for this section.